### DISCUSSION TITLE
Unable to query binding based on custom query

### DISCUSSION THREAD

**Discussion by hendrikjap** [Budibase/budibase#4415] [Help] [answered]
Hi guys,

I'm trying to bind the foreign key to my purchase id so I can display dynamic subtable based on the purchaseid. 
![image](https://user-images.githubusercontent.com/55825618/153421519-d9d2bc1a-9ba9-446a-ac05-51262001a10c.png)

![image](https://user-images.githubusercontent.com/55825618/153421906-31ea14df-a858-41f4-977a-dc741c04a4c6.png)

![image](https://user-images.githubusercontent.com/55825618/153421553-c85df55d-eead-40b5-a78f-d2f99d269103.png)

But there is no option to choose repeater.purchase.id in the handlebar formula.
![image](https://user-images.githubusercontent.com/55825618/153421574-a513fad7-5fa5-4f2a-91b0-9f46219a03d6.png)

Anything is wrong?

**Comment by hendrikjap** [comment, marked as answer]
I'm bumping this up

**Reply by mjashanks** [reply]
And so you should :)

It looks as though all of your bindings have disappeared. I feel like we have fixed something around this recently - https://github.com/Budibase/budibase/issues/4243 - have you updated Budibase in the last 2 weeks?

**Reply by hendrikjap** [reply]
No, my version is 1.0.44. I'll update it tonight. Thanks

**Reply by hendrikjap** [reply]
@mjashanks , it works! Thanks!

On side note, there seems a bug where I can't see the app version anymore 

![image](https://user-images.githubusercontent.com/55825618/153996820-7711d347-7d29-44b8-9617-0b6cfc290507.png)
